request/xhr.js

```diff
@@ -34,7 +34,9 @@ define([
 	has.add('native-xhr2-blob', function(){
 		if(!has('native-response-type')){ return; }
 		var x = new XMLHttpRequest();
-		x.open('GET', '/', true);
+		// URL should be fully-qualified as XMLHTTPRequest within a Worker
+		// created from a Blob does not support relative URL paths
+		x.open('GET', 'https://dojotoolkit.org/', true);
```

Comment:
ok, fair enough, please just explain that a bit better in the comment and then I'll accept. Thanks!